src/hooks/useCheckUpdates.tsx

```diff
@@ -8,53 +8,72 @@ import { ImportantMessageSetterContext } from "../context/importantMessage";
 const LAST_UPDATE_KEY = "LAST_UPDATE";
 const CHECK_UPDATE_INTERVAL_MINUTES = 60;
 
-export const useCheckUpdates = (): (() => void) => {
+type CheckUpdatesResult =
+  | "TOO_RECENT_CHECK"
```

Comment:
hahaha so `TOO_RECENT_CHECK` is like a default fallback value ah

Reply:
Hmmm, not really, the branch that leads to `TOO_RECENT_CHECK` occurs only when the last update was within 60mins.

Reply:
That reminds me, we wanted to add a condition to check every 30mins, i'll update this

Reply:
haha ok!